accounts/views.py

```diff
@@ -66,6 +69,32 @@ def account_profile(request):
     return render(request, "registration_login/account_settings.html", context)
 
 
+def contact(request):
+    form = ContactForm()
+    if request.method == 'GET':
+        context = {"form": form}
+        return render(request, "registration_login/contact_page.html", context)
+
+   
+    if request.method == "POST":
+        form = ContactForm(request.POST)
+
+        if form.is_valid():
+            msg_name = form.cleaned_data.get('name')
+            msg_email = form.cleaned_data.get('email')
+            msg_message = form.cleaned_data.get('message')
+
+        send_mail(
+            f'Kontakt od użytkownika {msg_name}',
+            f'Treść wiadomości: {msg_message}',
+            msg_email,
+            [settings.RECIPIENTS_LIST1, settings.RECIPIENTS_LIST2],
```

Comment:
W settingsach możesz listę podać, chodziło o to, że jeśli chcesz wartości maili ukrayć, dodać do .env to wtedy z enva musiałbyś wyciągać poszczególne emaile, ale w settingsach zadeklarujmy listę i tutaj już przekażmy listę